packages/material-ui/src/Modal/Modal.test.js

```diff
@@ -285,21 +288,25 @@ describe('<Modal />', () => {
 
       expect(onEscapeKeyDownSpy).to.have.property('callCount', 1);
       expect(onCloseSpy).to.have.property('callCount', 1);
+      expect(handleKeyDown).to.have.property('callCount', 0);
     });
 
     it('when disableEscapeKeyDown should call only onEscapeKeyDownSpy', () => {
```

Comment:
The test title is a bit confusing now we test handleKeyDown here too.

Reply:
Any idea for a better wording? Arguably, the wording for the props should be updated too

```
  /**
   * If `true`, clicking the backdrop will not fire any callback.
   */
  disableBackdropClick: PropTypes.bool,
```

Reply:
I'll have a think